dev/io.openliberty.io.opentelemetry/src/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -49,7 +49,7 @@ public static OpenTelemetry get() {
         return OpenTelemetry.noop();
     }
 
-    //GlobalOpenTelemetry cannot be set twice
+    //GlobalOpenTelemetry is only set by the Java Agent
     public static void set(OpenTelemetry openTelemetry) {
         throw new IllegalStateException(Tr.formatMessage(tc, "CWMOT5001.cannot.set.globalopentelemetry"));
     }
```

Comment:
The comment here is a bit mystifying - the `set` method clearly always throws an exception, so it's not clear how it would be set by the Java Agent.

Reply:
When using the Agent, this class is replaced entirely by the one provided by the agent, so nothing on this class is called.

Reply:
Would it be more appropriate to just say "GlobalOpenTelemetry cannot be set" 

This is addressing this problem: https://github.com/OpenLiberty/open-liberty/issues/25187#issuecomment-1760367683

Reply:
Ah, that makes sense.

Generally, comments should usually add some context to why things are like they are. It's obvious from looking at the code that `set()` will always throw an exception, but not why it's like that.

In this case, it's because we want users to inject the `OpenTelemetry` or `Tracer` object instead, so we should probably put something like this in the comment:

```
// GlobalOpenTelemetry cannot be set. The OpenTelemetry or Tracer object should be obtained via injection.

Aside: Looking at it again, I'm not sure why we didn't decide to delegate `get()` to get the `OpenTelemetry` object created by the mpTelemetry feature for the current application - possibly because at the time we were reliant on CDI to create that instance. Either way, we don't, we always return a no-op.